core/src/main/scala/cats/data/EitherT.scala

```diff
@@ -1187,13 +1187,13 @@ private[data] trait EitherTSemigroupK[F[_], L] extends SemigroupK[EitherT[F, L,
   implicit val F: Monad[F]
   def combineK[A](x: EitherT[F, L, A], y: EitherT[F, L, A]): EitherT[F, L, A] =
     EitherT(F.flatMap(x.value) {
-      case l @ Left(_)  => y.value
+      case _ @Left(_)   => y.value
       case r @ Right(_) => F.pure(r)
     })
 
   override def combineKEval[A](x: EitherT[F, L, A], y: Eval[EitherT[F, L, A]]): Eval[EitherT[F, L, A]] =
     Eval.now(EitherT(F.flatMap(x.value) {
-      case l @ Left(_)  => y.value.value
+      case _ @Left(_)   => y.value.value
```

Comment:
same change above? (move the left case below Right and use `case _ =>`

Reply:
✅